envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/StatusRouteTest.kt

```diff
@@ -0,0 +1,34 @@
+package pl.allegro.tech.servicemesh.envoycontrol
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import pl.allegro.tech.servicemesh.envoycontrol.config.EnvoyControlRunnerTestApp
+import pl.allegro.tech.servicemesh.envoycontrol.config.EnvoyControlTestConfiguration
+
+class StatusRouteTest : EnvoyControlTestConfiguration() {
+    companion object {
+        private val properties = mapOf(
+                "envoy-control.envoy.snapshot.routes.status.enabled" to true,
+                "envoy-control.envoy.snapshot.routes.status.pathPrefix" to "/my-status/",
+                "envoy-control.envoy.snapshot.routes.status.createVirtualCluster" to true
+        )
+
+        @JvmStatic
+        @BeforeAll
+        fun setupTest() {
+            setup(appFactoryForEc1 = { consulPort -> EnvoyControlRunnerTestApp(properties, consulPort) })
+        }
+    }
+
+    @Test
+    fun `should allow defining custom status prefix`() {
+        untilAsserted {
+            // when
+            val ingressRoot = callEnvoyIngress("/my-status")
+
+            // then
+            assertThat(ingressRoot.code()).isEqualTo(200)
```

Comment:
If we wrote a dedicated test for this functionality, there is no reason to not check if it actually works. By checking metric `vhost.secured_local_service.vcluster.status.upstream_rq_200`. It is all about this metric. The resposne status would be 200 even if we would have `status.enabled: false`, so it is not sufficient assertion.